### PR TITLE
some edits after a first review

### PR DESCRIPTION
**Pull request by agramfort**
cc @hichamjanati

### CODE REVIEW

File: groupmne/group_model.py [hunk 1]
```diff
@@ -25,17 +27,20 @@ def get_src_reference(subject="fsaverage", spacing="ico5", subjects_dir=None):
...
         src_ref = mne.read_source_spaces(fname_src)
-    elif os.path.exists(subjects_dir + subject):
+    elif os.path.exists(os.path.join(subjects_dir, subject)):
         src_ref = mne.setup_source_space(subject=subject,
                                          spacing=spacing,
                                          subjects_dir=subjects_dir,
                                          add_dist=False)
     else:
+        # XXX : I don't get the logic here. Why fetching
+        # fsaverage if subject is not fsaverage?
```
Comment (hichamjanati) [inline groupmne/group_model.py:43]: The idea here is: find a source space on disk and read it, if not on disk but fsaverage data is available then setup the src space if not fetch it and compute it

Comment (hichamjanati) [reply groupmne/group_model.py:43]: Ok I see now the confusion on the name of the subject ... this must be fixed

**Comment by hichamjanati** [comment]
Thanks for this review @agramfort. Shall we merge ?

**Comment by agramfort** [comment]
yes please merge and fix the remaining in a follow up PR. thx

**Pull request merged**